Tests/Model/RedirectUriTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Trikoder\Bundle\OAuth2Bundle\Tests\Model;
+
+use PHPUnit\Framework\TestCase;
+use RuntimeException;
+use Trikoder\Bundle\OAuth2Bundle\Model\RedirectUri;
+
+class RedirectUriTest extends TestCase
+{
+    public function testUris()
+    {
+        $passingUris = [
+            'http://www.oauth2.com/redirect/to-uri',
+            'https://oauth2.com/redirect-to-uri',
+            'com.example.test123:/redirect/to-uri',
+            'com.example.test:/redirect-to-uri',
+        ];
+
+        $faillingUris = [
+            'someRandomString',
```

Comment:
We should test mor failing URIs
```suggestion
            'someRandomString',
            'comexampletest:/redirect/to-uri',
            '.com.example.test:/redirect/to-uri',
            'com.example.test.:/redirect/to-uri',
            'com.example.test://redirect/to-uri',
```